modules/virt-must-gather-options.adoc

```diff
@@ -111,3 +117,14 @@ $ oc adm must-gather \
   --image=registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel9:v{HCOVersion} \
   /usr/bin/gather --images
 ----
+
+.InstanceTypes information
+
+The following command collects Instancetypes information from the cluster:
```

Comment:
Same comment as earlier:

> Should  `InstanceTypes` be in backticks?

Reply:
No backticks. The correct way to refer to them is actually instance types.